### PR TITLE
add/laurence-young

### PR DESCRIPTION
**Pull request by Yukiteno**
Added portfolio for Laurence Young.
Thanks!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,8 +4,8 @@ A list of developer portfolios for your inspiration
 
 Have you built a portfolio? Are you proud of it?! Open a [PR](./CONTRIBUTING.md) to this repo and let's showcase your work! Refer to the [CONTRIBUTING](./CONTRIBUTING.md) file for direction.
 
-This repo was inspired by [Ali Spittel's](https://twitter.com/ASpittel) tweet
-[<img width="597" alt="Portfolio" src="https://user-images.githubusercontent.com/7671983/64871043-bab42880-d644-11e9-8e87-4a98d06339c9.png">](https://twitter.com/ASpittel/status/1171604728951779328)
+This repo was inspired by [Ali Spittel&#39;s](https://twitter.com/ASpittel) tweet
+[`<img width="597" alt="Portfolio" src="https://user-images.githubusercontent.com/7671983/64871043-bab42880-d644-11e9-8e87-4a98d06339c9.png">`](https://twitter.com/ASpittel/status/1171604728951779328)
```
Comment (eclectic-coding) [inline README.md:8]: Please remove these changes

File: README.md [hunk 2]
```diff
@@ -134,7 +134,7 @@ Hopefully this repo can serve as a source of inspiration for your portfolio!
 - [Christian Toscano](https://achris.me)
 - [Chuck Smith](https://chucksmith.dev)
 - [Chuckz Okoye](https://chuckzokoye.com)
-- [Clyde D'Souza](https://clydedsouza.net)
+- [Clyde D&#39;Souza](https://clydedsouza.net)
```
Comment (eclectic-coding) [inline README.md:137]: Please remove these changes

**Review by eclectic-coding — COMMENTED** [review]
Please updat as noted and I will add your site.

**Comment by Yukiteno** [comment]
> Please updat as noted and I will add your site.

Hi! I'm very sorry for this, I think it's caused by one of my extension. I just removed these changes.

**Comment by eclectic-coding** [comment]
No worries it happens to us all.

**Pull request merged**